src/cbadc/utilities.py

```diff
@@ -232,6 +232,58 @@ def read_byte_stream_from_url(
             pass
     raise StopIteration
 
+def csv_2_control_signal(filename: str, M: int, msb2lsb: bool = False, separator: str = ','):
+    """Creates an iterator that reads a control sequence from a CSV file.  
+
+    Parameters
+    ----------
+    filename : `str`
+        filename for input file
+    M : `int`
+        number of controls
+    msb2lsb : `bool`
+        bit order of input file.
+        Default is least significant to most significant bit from left to right (LSB to MSB).
```

Comment:
Default is least significant bit (LSB) to most significant bit (MSB) from left to right.

Reply:
yes, that sounds a bit clearer